requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Version updates managed by dependabot
 
 apypie==0.7.1
-broker[docker,podman,hussh]==0.6.12
```

Comment:
I anticipate this dependency group change might impact our CI execution as well, since we support both hussh and ssh2_python ssh backends there for component testing, do you intend to change that logic and force only ssh2_python backend now?